.gitignore

```diff
@@ -67,3 +67,4 @@ yarn-error.log
 .pnp.js
 # Yarn Integrity file
 .yarn-integrity
+packages/gatsby-theme-nodeschool/src/default-language.json
```

Comment:
I don't think this is needed anymore since we are not using the random file

Reply:
Thanks, I forgot it